src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/Worker.java

```diff
@@ -94,7 +94,7 @@
 
     // Default configs for periodic shard sync
     private static final int SHARD_SYNC_SLEEP_FOR_PERIODIC_SHARD_SYNC = 0;
-    private static final int MAX_INITIALIZATION_ATTEMPTS = 20;
+    private static final int MAX_INITIALIZATION_ATTEMPTS = 2;
```

Comment:
I'm assuming this was changed temporarily to make the tests run faster (I did the same thing). With the new lease table wait time upon initialization, we should maybe sync up on making this a config like it is in 2.x. That we we can mock it to be 1 or 2 for unit tests. Doesn't need to be in scope for this PR.

Reply:
Oops, forgot to change this back. :P  Yeah, now that unit test is running so long! We can sync up on this.